src/routes/home/Home.jsx

```diff
@@ -1,18 +1,34 @@
 import { Outlet, Link, useLocation } from "react-router-dom";
-import { useState } from "react";
+import React, { useState } from "react";
 import dataOfwallet from "../../data/wallet";
 import { HomeContainer, TabNav, TabButton } from "./Home.style";
 
 const VM = "vm";
 const WALLET = "wallet";
 
+export const WalletContext = React.createContext();
+export const VendingMachineContext = React.createContext();
+
 function Home() {
     const { pathname } = useLocation();
-    const [currentTab, setCurrentTab] = useState(pathname.slice(1));
     const [record, setRecord] = useState([]);
     const [moneyInVendingMachine, setMoneyInVendingMachine] = useState({});
     const [wallet, setWallet] = useState(dataOfwallet);
 
+    const currentTab = (() => {
```

Comment:
즉시 실행 함수도 괜찮지만 해당 로직은 요렇게 바꿀 수 있을 것 같아요!
```suggestion
const currentTab = pathname.slice(1) || null;
```